### PR TITLE
ecaps2 (XEP-0390) to version 0.1

### PR DESCRIPTION
**Pull request by horazont**
Now that it’s experimental, here come the minor changes I had in the pipeline.

* Some editing
* Explicitly mention the possible need for rate-limiting of caps processing in Security Considerations
* Reference UTF-8 and Base64 RFCs
* Reference U+002E properly

### CODE REVIEW

File: xep-0390.xml [hunk 1]
```diff
@@ -316,9 +323,9 @@
 000001d9</code>
     <p>Running this octet string through the hash functions leads as to the following &hashset;:</p>
     <code><![CDATA[
-<c xmlns="urn:xmpp:caps" xmlns:hashes="urn:xmpp:hashes:2">
-  <hashes:hash algo="sha-256">kzBZbkqJ3ADrj7v08reD1qcWUwNGHaidNUgD7nHpiw8=</hashes:hash>
-  <hashes:hash algo="sha3-256">79mdYAfU9rEdTOcWDO7UEAt6E56SUzk/g6TnqUeuD9Q=</hashes:hash>
+<c xmlns="urn:xmpp:caps">
+  <hash xmlns="urn:xmpp:hashes:2" algo="sha-256">kzBZbkqJ3ADrj7v08reD1qcWUwNGHaidNUgD7nHpiw8=</hashes:hash>
```
Comment (linkmauve) [inline xep-0390.xml]: The closing tag still uses a prefix, like in the previous revision.

Same for every other case.

**Comment by horazont** [comment]
@linkmauve Nice catch! Fixed.

File: xep-0390.xml [hunk 2]
```diff
@@ -51,6 +52,12 @@
     <jid>jonas@wielicki.name</jid>
   </author>
   <revision>
+    <version>0.1</version>
+    <date>2017-03-23</date>
+    <initials>jwi</initials>
+    <remark><p>Add security consideration for caps spamming.</p><p>Properly reference UTF-8 and Base64 RFCs.</p><p>Remove namespace prefixes from examples.</p><p>Properly reference &fullstop;.</p><p>Clarify the first error condition in the Hash Function Input algorithm.</remark>
```
Comment (linkmauve) [inline xep-0390.xml]: You forgot the last </p> here.  Also you should probably split them on multiple lines, and use <ul/> and <li/>.

**Comment by horazont** [comment]
@linkmauve Fixed

**Comment by SamWhited** [comment]
Merging; sorry for the long delay. Now that the editor infrastructure is more or less back I'm catching up.

**Pull request merged**